openedx/core/lib/xblock_serializer/data.py

```diff
@@ -1,12 +1,11 @@
 """
 Simple data structures used for XBlock serialization
 """
-from __future__ import annotations
-from typing import NamedTuple
+from typing import Optional
 
-
-class StaticFile(NamedTuple):
+class StaticFile:
     """ A static file required by an XBlock """
-    name: str
-    url: str | None
-    data: bytes | None
+    def __init__(self, name: str, url: Optional[str] = None, data: Optional[bytes] = None):
+        self.name = name
+        self.url = url
+        self.data = data
```

Comment:
Sphinx AutoAPI is failing for NamedTuple here so replacing it with a standard Python class.

Fixes: `exception: 'list' object has no attribute 'get_children'`